tests/Doctrine/Tests/ORM/Tools/Console/Command/ValidateSchemaCommandTest.php

```diff
@@ -57,6 +57,11 @@ public function testNotInSync(): void
 
     public function testNotInSyncVerbose(): void
     {
+        $schemaManager = $this->createSchemaManager();
+        if ($schemaManager->tablesExist('cache_login')) {
```

Comment:
This is also a mess. This test will fail if SQLite starts reporting that it supports foreign keys _and_ if run as part of some other tests. The `cache_login` table will already exist at this point and won't be part of the diff.